packages/web/pages/index.tsx

```diff
@@ -99,9 +103,9 @@ const Landing: React.FC = () => {
     <>
       <HeadComponent
         title="MetaGame: A Massive Online Coordination Game!"
-        description="To find your metagame means to play life in the optimal way. Coordinating with others on building a better world; doing things that create a a positive impact make you happy AND earn you money."
+        description="To find your metagame means to play life in the optimal way. Coordinating with others to build a better world; make a positive impact, make you happy AND earn you money."
```

Comment:
```suggestion
        description="To find your metagame means to play life in the optimal way. Coordinating with others to build a better world; make a positive impact, make you happy, 𝘈𝘕𝘋 earn you money."
```

Reply:
Don't like the Oxford comma?

Reply:
Its not that, I had already updated the text to use `&amp;` in an effort to reduce the char count from 190 to the recommended limit of 160 but forgot to add in the oxford comma. 🤦🏻

Reply:
✅